src/scss/components/cta_uids3.scss

```diff
@@ -0,0 +1,121 @@
+@import '../abstracts/variables';
```

Comment:
Why does this file have `_uids3` in the name when other files being moved over do not?

Reply:
We need it there until https://github.com/uiowa/uiowa/issues/8234.  I'm fine with any other name, but there's already a newer `cta.scss`.